request-forms/README.md

```diff
@@ -0,0 +1,144 @@
+    loanAmount: '10000',
+    email: 'test-dev@moffin.mx',
+    firstName: 'Prueba',
+    phone: '+52 <your phone number>', // Use your own phone number if you wish to test the send-sms endpoint later
+    rfc: 'PRPU800101111',
+    curp: 'PETJ970221HJCRRN09',
+    birthdate: '1997-01-01',
+    address: 'Jaime Balmes',
+    city: 'Cd de Mexico',
+    state: 'CMX',
+    zipCode: '11510',
+    exteriorNumber: '8',
+    neighborhood: 'Los morales',
+    country: 'MX',
+    nationality: 'MX'
+  })
+  console.log('Created Form ID', form.formId)
+```
+
+If the Form is created without issues, you'll get the form ID back, we will use that ID for the remaining requests.
+
+
+### Update a Form
+This endpoint uses PATCH to update an existing form (before submission), you can use it to provide edit functions in your app.
+
+```js
+ await client.patch(`/form/${form.formId}`, {
+    firstLastName: 'Apellido'
+ })
+```
+
+### Send SMS/Email
+These two endpoints send the NIP associated with a form to the applicant's email or sms. You can use them to verify that the email and/or phone indeed belong to the applicants. At the moment you only need to verify either one.
+
+```js
+ // Send SMS 
+ const { data: smsData } = await client.post(`/form/nip/send-sms/${form.formId}`, {
+    organizationName: 'Moffin' // Name to display in the SMS message
+ })
+ console.log('SMS Response Body', smsData)
+ 
+ // Send Email (similar to previous endpoint)
+ const { data: emailData } = await client.post(`/form/nip/send-email/${form.formId}`, {
+    organizationName: 'Moffin' // Name to display in the Email
+ })
+ console.log('Email Response Body', smsData)
+```
+
+Both Email and SMS return some data, than can be used to check if the SMS or Email was sent without issues, as well as the remaining attempts for that code. Keep in mind that our API only allows sending one message every 2 minutes, so this endpoint may fail if you use it multiple times for the same form.
+
+```json5
+// SMS Response Body 
+{
+  timeUntilNewAttempt: '2022-10-12T17:41:56.185Z',
+  remainingAttempts: 2,
+  sent: true
+}
+```
+
+```json5
+// Email Response Body 
+{
+  timeUntilNewAttempt: '2022-10-12T17:41:56.185Z',
+  remainingAttempts: 2,
+  sent: true
+}
+```
+
+### Verify NIP
+You can use this endpoint to verify the NIP sent by the previous endpoints, keep in mind this endpoint will return 204 if the verification is succesful, not 200.
+
+```js
+ await client.post(`/form/nip/validate/${form.formId}`, {
+    nip: code // code is the NIP received by the user
+ })
+```
+
+### Submit the Form
+Once the form has been populated with the correct information, and the it has been verified, you can submit the form to our system using this endpoint. Make sure to request all the required fields when creating the form (provided in the Create Form example).
+
+```js
+ await client.post(`/form/submit/${form.formId}`, {
+    validateNIPAuthentication: true // Ensure the NIP was validated
+ })
+```
```

Comment:
También agregaría un comentario explicando para que es el campo `validateNIPAuthentication` y cuando usarlo.

Also así:

_If you have built your own solution to validate NIP outside Moffin, you can set the flag `validateNIPAuthentication` as `false` to disable the validation. If you set this flag as true, we will check if at least the phone number or email has been validated before the form submission, if none is validate we will return an error `403`_